.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+name: Handle Release
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+        
+      - name: Set GOPATH
+      # temporary fix
+      # see https://github.com/actions/setup-go/issues/14
+        run: |
+          echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+          echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+        shell: bash
+    
+      - name: Prepare environment
+        run: |
+          go mod download
```

Comment:
I think (and maybe I'm wrong) you'll need && between these lines. Otherwise a command will fail but next one will continue to execute.

Reply:
From docs:
> Each run keyword represents a new process and shell in the runner environment. When you provide multi-line commands, each line runs in the same shell.

Seems not needed.

Reply:
Without seeing proof, I won't be convinced :) If they run in the same shell, it most likely means they're separated with `;` and not `&&`. Try

```
false
date
```

you should be seeing a failure not the date.

Reply:
@RubenBaez have you checked how github handles multi-line scripts?

Reply:
@RubenBaez Btw, I tried this out and it looks like @ahmetb was correct. If one line in a multi-line run script fails, the following lines _will be exectued regardless_.